app/javascript/gobierto_data/webapp/components/commons/LinkableText.vue

```diff
@@ -21,6 +25,10 @@ export default {
       url: {
         type: String,
         default: ''
+      },
+      fontSize: {
```

Comment:
I think this is not necessary. Since the `<a>` or `<span>` are complementary top-level selectors, you can set the style straightly in the parent:

```vue
<LinkableText
  :text="text"
  :url="url"
  :style="{ fontSize }"
/>
```

Same for `class`. If you can avoid to add props, do it. Keep it simple

Reply:
Done!